### PR TITLE
add reward-sdk

### PR DESCRIPTION
**Pull request by mul53**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "@apollo/client": "^3.3.7",
+    "@fuseio/rewards-sdk": "file:../rewards-sdk",
```
Comment (leonprou) [inline package.json:36]: let's use the SDK

**Review by leonprou — CHANGES_REQUESTED** [review]
Looks good. see the one comment

File: package.json (hunk 1 above)
Comment (mul53) [reply package.json:36]: installing it now

**Comment by mul53** [comment]
this is good to go now

**Pull request merged**